asdf_astropy/resources/manifests/astropy-1.2.0.yaml

```diff
@@ -6,15 +6,15 @@ description: |-
   model classes, which are handled by an implementation of the ASDF
   transform extension.
 asdf_standard_requirement:
-  gte: 1.6.0
```

Comment:
Note to reviewers this manifest requires ASDF standard 1.6.0 which has not yet become the stable release. Since this PR adds new features that are compatible with the current stable ASDF standard 1.5.0 I chose to change this manifest (`astropy-1.2.0`) to instead support the stable version (using all the pre 1.6.0 schemas) and adding the new WCS schemas.

I also added `astropy-1.3.0` which support 1.6.0 and also adds the new WCS schemas for that standard version.

So to summarize. Before this PR:
- `astropy-1.2.0` supported ASDF standard 1.6.0 (the unused development version)

After this PR:
- `astropy-1.2.0` supports ASDF standard 1.5.0 and adds the WCS schemas
- `astropy-1.3.0` supports ASDF standard 1.6.0 and adds the WCS schemas